app/build.gradle

```diff
@@ -62,13 +62,19 @@ dependencies {
       'androidx.appcompat:appcompat:1.0.2',
       'androidx.constraintlayout:constraintlayout:1.1.3',
       'androidx.core:core-ktx:1.0.2',
+      'androidx.legacy:legacy-support-v4:1.0.0',
+      'androidx.lifecycle:lifecycle-runtime-ktx:2.2.0-alpha03',
+      'androidx.lifecycle:lifecycle-extensions:2.0.0',
```

Comment:
@nikitamarysolomanpvt I believe this and above line are still not in alphabetical order.

Reply:
Done